cli/src/main/groovy/io/micronaut/cli/profile/repository/MavenProfileRepository.groovy

```diff
@@ -139,6 +140,26 @@ class MavenProfileRepository extends AbstractJarProfileRepository {
         }
     }
 
+    @CompileDynamic
+    protected String getMavenLocal() {
+        if(!mavenLocalLocation) {
+
+            File settingsXml = new File(System.getProperty("user.home"), "/.m2/settings.xml")
+            if(settingsXml.exists()) {
+                String localRepo = new XmlSlurper().parseText(settingsXml.text)?.localRepository
+                if(localRepo) {
+                    mavenLocalLocation = "${localRepo.replace('${user.home}', System.getProperty("user.home"))}"
+                }
+            }
+
+            if(!mavenLocalLocation) {
+                mavenLocalLocation = "${System.getProperty("user.home")}/.m2/repository/"
+            }
+        }
+
+        mavenLocalLocation
+    }
+
```

Comment:
Could you separate out the CLI changes into a separate pull request? They seem unrelated to the Route53 changes.

Reply:
hmm there shouldn't be any changes here.. when I compare this file with master it says 100% identical.